src/components/vehicleDataGroup/index.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import VehicleDataGroupField from './VehicleDataGroupField.tsx';
+import { CloseIcon } from '../icons';
+
+import VehicleDataGroup from './';
+
+<Meta
+  title="Components/Data Display/VehicleDataGroup"
+  component={VehicleDataGroup}
+/>
+
+export const ItemLabel = () => (
+    <>
+    <CloseIcon />
+    Some value
+    </>
+)
+
+export const ItemValue = () => (
+  <>
+    <CloseIcon />
+    Some value
+  </>
+)
+
+export const Template = (args) => {
+
+const label = 'VehicleDataGroupField label';
+const value = 'VehiculeDataGroupField Value';
+
+return (
+
+<VehicleDataGroup {...args} title={args.title} w={args.w}>
+  <VehicleDataGroupField label={label} value={value} />
+  <VehicleDataGroupField label={label} value={value} />
+  <VehicleDataGroupField label={label} value={value} />
+  <VehicleDataGroupField label={<ItemLabel />} value={<ItemValue />} />
+</VehicleDataGroup>
+) };
+
+# GroupLabel
+
+## Overview
+
+<Canvas>
+  <Story
+    name="GroupLabel"
+    args={{
+      title: 'Vehicle Group Data',
+      w: '400px',
```

Comment:
We have some patterns defined, the general idea is to avoid random sizes and use the containers and sized that have been defined to keep the desings consistent. Maybe you can check with the designer the intention they have for the compenet and what sizes we need for the use cases.

https://main.components-pkg.branch.autoscout24.ch/?path=/story/foundations-theme--containers